src/app/ipv/middleware.js

```diff
@@ -210,6 +211,47 @@ function checkJourneyAction(req) {
   }
 }
 
+// getCoiUpdateDetailsJourney determines the next journey based on the detailsToUpdate
+// field of the update-details page
+function getCoiUpdateDetailsJourney(detailsToUpdate) {
+  if (!detailsToUpdate) {
+    return;
+  }
+  // convert to array if its a string
+  if (typeof detailsToUpdate === "string") {
+    detailsToUpdate = [detailsToUpdate];
+  }
+
+  if (detailsToUpdate.includes("cancel")) {
+    return UPDATE_DETAILS_JOURNEY_TYPES.UPDATE_CANCEL;
+  }
+  // send to update-names-dob journey if dateOfBirth selected
+  if (detailsToUpdate.includes("dateOfBirth")) {
+    return UPDATE_DETAILS_JOURNEY_TYPES.UPDATE_NAMES_DOB;
```

Comment:
I think this actually wants to be a `return saveSessionAndRedirect()` call, we want to send the user to the contact us webpage, which we usually handle on the front-end and not bother sending it to the backend.

Think it's fine to handle this in the `handleJourneyEvent()` method and not here directly, but we definitely want to avoid sending this to the backend I would think

Reply:
ah ok, great I was trying to figure out how to do that

Reply:
@RogVanGDS according to https://govukverify.atlassian.net/browse/PYIC-5809 it goes to another form (update-name-date-birth) where the user is presented with another option to either go back or continue to the contact us page, so maybe better to let the backend handle this so it knows where to send the user if they select 'go back and check your details again'

Reply:
Yep, I think I agree @sam803